streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -406,24 +361,6 @@ private void cleanGlobal(final String intermediateUserTopic) {
         Assert.assertEquals(0, exitCode);
     }
 
-    private void waitUntilUserTopicsAreDeleted() {
```

Comment:
Thank @guozhangwang. Now I got it.

The conditions are slightly different for both cases: here we check the remaining topic names (as we don't know the names of the internal topics). If we use `TestUtils.waitForCondition(TopicsDeletedCondition)` we would first need to get the internal topic names that we expect be deleted -- IMHO, this would make it more complex than it is atm. I would prefer to keep the PR as-is.